owncloud/test/test.py

```diff
@@ -601,7 +601,7 @@ def test_share_with_user(self, file_name):
         self.assertTrue(isinstance(share_info, owncloud.ShareInfo))
         self.assertEquals(share_info.get_path(), path)
         self.assertTrue(type(share_info.get_id()) is int)
-        self.assertTrue(share_info.get_permissions(), '31')
+        self.assertEquals(share_info.get_permissions(), '31')
```

Comment:
@nickvergessen 
Yeah definitely weird. Thats what made me thought it wasnt error at first since they don't look copy pasted. Both should be int, right? I'll fix it in a sec